Resideo_eshopping/resideo_eshopping/lib/util/dbhelper.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:resideo_eshopping/model/eshopping_model.dart';
+import 'package:sqflite/sqflite.dart';
+import 'package:path_provider/path_provider.dart';
+import 'dart:io';
+import 'dart:async';
+
+class Dbhelper
+{
+
+  static final Dbhelper _helper=Dbhelper.private();
+
+  String tblname='product';
+  String colid="id";
+  String coltitle="title";
+  String cols_desc="s_desc";
+  String colimg="img";
+  String colprice="price";
+  String colquantity="quantity";
+  String coll_desc="l_desc";
+  String colcategory="category";
+  String colrating="rating";
+  String colreview="review";
+
+  Dbhelper.private();
+
+  factory Dbhelper()
+  {
+    return _helper;
+  }
+
+  Database _db;
+
+  Future<Database> get db async{
+     if(_db==null)
+     _db=await initializedb();
+     return _db;
+  }
+
+  Future<Database> initializedb() async
+  {
+    Directory dir= await getApplicationDocumentsDirectory();
+    String path=dir.path+'eshoppingdb.db';
```

Comment:
Please declare variables with db name and version and use those variables here instead of hard coding the values